docs/src/data.md

```diff
@@ -0,0 +1,61 @@
+# Model data and the `Data` trait
+
+The heart of a `druid` application is your application model. Your model drives
+your UI. When you mutate your model, druid compares the old and new version,
+and propogates the change to the components ('widgets') of your application that
+are effected by the change.
+
+For this to work, your model must implement the `Clone` and `Data` traits. It
+is important that your model be cheap to clone; we encourage the use of `Arc`
+pointers to allow cheap cloning of more expensive types.
+
+The `Data` trait has a single method:
+
+```rust,no_run,noplaypen
+{{#include ../../druid/src/data.rs:same_fn}}
```

Comment:
I assume this is supposed to bring in the doc string? https://github.com/xi-editor/druid/blob/fdd845c80599c9d908b22e69762952f3b939a1ff/docs/src/data.md suggests this will not happen

Reply:
It works when you run it with mdbook